### PR TITLE
issue: #3133 updated the path in code and also added the file named c…

### PR DESCRIPTION
**Pull request by Cretus-iosDev**
…lusterstoragecontainer.yaml

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Type of changes**
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Feature/Issue validation/testing**:

Please describe the tests that you ran to verify your changes and relevant result summary. Provide instructions so it can be reproduced.
Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

- Logs

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Checklist**:

- [ ] Have you added unit/e2e tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

**Release note**:

```release-note

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -80,7 +80,7 @@ deploy: manifests
 	${KUSTOMIZE} build config/default | kubectl apply -f -
 	kubectl wait --for=condition=ready pod -l control-plane=kserve-controller-manager -n kserve --timeout=300s
 	sleep 2
-	${KUSTOMIZE} build config/runtimes | kubectl apply -f -
```
Comment (greenmoon55) [inline Makefile:83]: Maybe put it in `config/storagecontainers`?

**Review by greenmoon55 — CHANGES_REQUESTED** [review]
Maybe put it in config/storagecontainers? Can you also update the description of the PR?

**Comment by Cretus-iosDev** [comment]
i'll make the changes

**Pull request closed without merging**